docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,27 @@
+## Deployment
+This document covers steps on setting up this repository on various cloud hosting providers.
+
+  - [AWS](#aws)
+    - [Pre-requisites](#pre-requisites)
+    - [Deployment steps](#deployment-steps)
+
+### AWS
+
+#### Pre-requisites
+- Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+- Set up `plio-s3-bot` on your local.
```

Comment:
Should we add a note that either they should get the credentials from the admins of Plio or if you are setting up your own instance, add your AWS details?

Reply:
I think probably we need a better guideline of what this bot does and how to set it up.

I think as an open-source project, we should always assume that people will be using this guide to set up their own infrastructure/installation. If there's someone joining the Plio team and managing Plio's infrastructure, there should be an internal document.